hugegraph-core/src/main/java/org/apache/hugegraph/config/CoreOptions.java

```diff
@@ -635,6 +635,14 @@ public static synchronized CoreOptions instance() {
                     2
             );
 
+    public static final ConfigOption<String> NODE_EXTERNAL_URL =
+            new ConfigOption<>(
+                    "server.role.node_external_url",
+                    "The url of external accessibility",
+                    disallowEmpty(),
+                    "127.0.0.1:8080"
```

Comment:
we can pass it from restserver config to graph config, like rpc url

Reply:
I'm a little confused about the question raised
now, I put the config to resetserver, but the stateMachine put in graph instance, so these config must be copy to graph config